project/src/components/review-form/review-form.tsx

```diff
@@ -0,0 +1,58 @@
+import React, { useState, ChangeEvent } from 'react';
```

Comment:
```suggestion
import { useState, ChangeEvent } from 'react';
```
теперь можно не импортировать React целиком - он по договоренности
https://reactjs.org/blog/2020/09/22/introducing-the-new-jsx-transform.html